components/Footer.tsx

```diff
@@ -0,0 +1,157 @@
+import Link from 'next/link'
+import en from '../locales/en'
+import fr from '../locales/fr'
+import Image from 'next/image'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { icon } from '../lib/loadIcons'
+import logo from '../public/wmms-blk.svg'
+
+interface links {
```

Comment:
Links should be uppercase here to be consistent

Reply:
Good catch